sssd_test_framework/roles/ad.py

```diff
@@ -360,6 +363,34 @@ def test_example(client: Client, ad: AD):
         """
         return ADComputer(self, name, basedn)
 
+    def dns(self) -> ADDNSServer:
+        """
+        Get DNS server object.
+
+            Get methods use dig and is parsed by jc. The data from jc contains several nested dict,
+            but two are returned as a tuple, ``answer, authority``.
+
+        .. code-block:: python
+            :caption: Example usage
+
+            # Create forward zone, add and get record and assert ip
+            forward_zone = ad.dns().zone("smb.test").create()
+            forward_zone.add_record("client", "172.16.200.5")
+            result = forward_zone.get_record("client.smb.test")
+            assert result[0].get("data") == "172.16.200.5"
```

Comment:
Is there a way to shortcut accessing the record values with  `result.data`, or `result.ttl` instead of   `result[0].get("data")` ??

As a test writer this would be much cleaner to me.

            assert result.data == "172.16.200.5"